src/basic/chase.c

```diff
@@ -77,6 +77,24 @@ static int log_prohibited_symlink(int fd, ChaseFlags flags) {
                                  strna(n1));
 }
 
+static int openat_opath_with_automount(int dir_fd, const char *path, bool automount) {
+        static int can_open_tree = true;
```

Comment:
I'm wondering if a process couldn't do some user namespace shenanigans mid runtime to suddenly gain the privileges necessary to use open_tree() where it previously couldn't.

Reply:
The operation is not privileged. So, I do not think it is possible...